package.json

```diff
@@ -24,6 +24,7 @@
     },
     "dependencies": {
         "@astrojs/mdx": "^2.3.1",
+        "@octopus/docs": "file:",
```

Comment:
@twerthi do you know what this is for? I thought it was maybe an IDE auto addition that we don't need?